common/match/pool.ts

```diff
@@ -193,6 +210,30 @@ const _pools = [
                 };
             }
 
+            if (toggles.includes('pupil-screening-success')) {
+                query.pupil_screening = {
+                    some: {
+                        invalidated: false,
+                        status: 'success',
+                    },
+                };
+            }
+
+            if (toggles.includes('pupil-screening-pending')) {
+                query.pupil_screening = {
+                    some: {
+                        invalidated: false,
+                        status: 'pending',
+                    },
+                };
+            }
+
+            if (toggles.includes('pupil-screening-unknown')) {
```

Comment:
Hmm, I think we also want to only request pupil screenings for pupils who do not have a pending interest confirmation

Reply:
I'll create a followup PR for that